lib/rest-to-tx-converter.js

```diff
@@ -106,4 +105,19 @@ RestToTxConverter.prototype.convert = function(payment, callback) {
 
 };
 
+/**
+ *  Convert a numerical transfer rate in ripple-rest format to ripple-lib
+ *
+ *  @param {Number|String} transferRate
+ *
+ *  @returns {Number|String} numbers will be converted while strings are returned
+ */
+RestToTxConverter.prototype.convertTransferRate = function(transferRate) {
```

Comment:
Can you add a piece of documentation so we don't forget why this is the way it is?

A fee of 1% requires 101% of the destination to be sent for the destination to receive 100%. The transfer rate is specified as the input amount as fraction of 1 billion. To specify the default rate of 0%, a 100% input amount, specify 1000000000. To specify a rate of 1%, a 101% input amount, specify 1010000000.

From the (wiki)[https://wiki.ripple.com/Set_a_transfer_rate}